diff_cover/report_generator.py

```diff
@@ -363,6 +364,11 @@ class StringReportGenerator(TemplateReportGenerator):
     """
 
     TEMPLATE_NAME = "console_coverage_report.txt"
+    INCLUDE_SNIPPETS = False
```

Comment:
IMO this format implies this is a constant and this very much is not.

I think this should just be a property on the object itself.

```
def __init__(self, violations_reporter, diff_reporter, show_uncovered=False):
        super().__init__(violations_reporter, diff_reporter)
        self.include_snippets = show_uncovered
```

Reply:
The attribute is now in lowercase as well as the other twos (to keep it consitent)